src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -76,6 +76,7 @@ private function addGlobalsSection(ArrayNodeDefinition $rootNode)
                     ->useAttributeAsKey('key')
                     ->example(array('foo' => '"@bar"', 'pi' => 3.14))
                     ->prototype('array')
+                        ->normalizeKeys(false)
```

Comment:
Don't we have the same issue in 2.7 as well? I'm wondering if it would not be "safer" to do this in master instead.

Reply:
We merged [a similar fix](https://github.com/symfony/symfony/pull/21718) as an assumed BC break on master only. This one may have worse impacts. I don't think it is worth breaking existing apps relying on this behavior, so I'd do it on master only (and mention it in the `UPGRADE-4.1.md` file).

Reply:
For the record, a [similar PR](https://github.com/symfony/symfony/pull/25049) has been rejected on the workflow component to keep the BC. But in the workflow there was an alternative way to configure things.

Reply:
Sure, one person’s bugfix is another person's BC break. Relying (and keeping) a behavior where the keys of an array magically change doesn't sound like a good idea to me though.